gcampus/tools/models/calibration.py

```diff
@@ -0,0 +1,89 @@
+#  Copyright (C) 2022 desklab gUG (haftungsbeschränkt)
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Affero General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Affero General Public License for more details.
+#
+#  You should have received a copy of the GNU Affero General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__ALL__ = ["MeasurementKit", "Calibration"]
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from gcampus.core.models.parameter import ParameterType
+
+
+class MeasurementKit(models.Model):
```

Comment:
Maybe put the `MeasurementKit` model in its own file called `kit.py`. Maybe we want to add more functionality to this model other than the calibration.